### PR TITLE
2.x Update Magnes

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - This PR applies a fix for the Hostname Verifier issue in Braintree `2.x`

 ### Checklist

 - [x] Added a changelog entry

### Authors

- @Epreuve
- @sshropshire
- @sarahkoop

### CODE REVIEW

**Review by demerino — APPROVED** [review]
👍

File: PayPalOneTouch/src/main/java/com/paypal/android/sdk/onetouch/core/sdk/AppSwitchHelper.java [hunk 1]
```diff
@@ -29,13 +29,13 @@
 public class AppSwitchHelper {
 
     private static final String WALLET_APP_PACKAGE = "com.paypal.android.p2pmobile";
-    private static final String WALLET_APP_CERT_SUBJECT = "O=Paypal";
-    private static final String WALLET_APP_CERT_ISSUER = "O=Paypal";
-    private static final int WALLET_APP_PUBLIC_KEY_HASH_CODE = 34172764;
+
+    public static final String PAYPAL_BASE_64_ENCODED_SIGNATURE1 = "tFPTfHgoVveCqOCfFO4guE9JoxB19H/ToHmC2Mr+/9k=\n";
+    public static final String PAYPAL_BASE_64_ENCODED_SIGNATURE2 = "x8YuoPbi9uuof5VAaYdQVTDvL1FufN5ZkdcUAzFEgHI=\n";
```
Comment (sarahkoop) [inline PayPalOneTouch/src/main/java/com/paypal/android/sdk/onetouch/core/sdk/AppSwitchHelper.java:34]: can these be private?

**Review by sarahkoop — APPROVED** [review]
One public/private change suggestion otherwise looks good to me

File: PayPalOneTouch/src/main/java/com/paypal/android/sdk/onetouch/core/sdk/AppSwitchHelper.java (hunk 1 above)
Comment (sshropshire) [reply PayPalOneTouch/src/main/java/com/paypal/android/sdk/onetouch/core/sdk/AppSwitchHelper.java:34]: Oh yeah good catch, gonna update these.

**Pull request merged**